deepchem/feat/__init__.py

```diff
@@ -73,6 +73,12 @@
 except ModuleNotFoundError:
   pass
 
+try:
```

Comment:
I think we can pass all HF-related imports under one try-catch for cleanliness here